test_TS3Connection.py

```diff
@@ -17,39 +17,59 @@ def setUp(self) -> None:
         self.conn = MockTS3Connection()
 
     def test_parse_resp_left_event(self):
-        resp = b"notifyclientleftview cfid=1 ctid=0 reasonid=8 " \
-               b"reasonmsg=Left. clid=1"
+        resp = (
+            b"notifyclientleftview cfid=1 ctid=0 reasonid=8 " b"reasonmsg=Left. clid=1"
```

Comment:
This does not seem right, this should be changed into a single byte string.

Reply:
The behaviour shouldn't change due to this change.

Both versions should result in the same behaviour as you can see below:

```
>>> resp = b"notifyclientleftview cfid=1 ctid=0 reasonid=8 " \
...                b"reasonmsg=Left. clid=1"
>>> 
>>> print( resp )
b'notifyclientleftview cfid=1 ctid=0 reasonid=8 reasonmsg=Left. clid=1'
>>> 
```

```
>>> resp = (
...             b"notifyclientleftview cfid=1 ctid=0 reasonid=8 " b"reasonmsg=Left. clid=1"
... )
>>> 
>>> print( resp )
b'notifyclientleftview cfid=1 ctid=0 reasonid=8 reasonmsg=Left. clid=1'
>>> 
```

Alternativley, we can also write it like this:

```
>>> resp = b"notifyclientleftview cfid=1 ctid=0 reasonid=8 reasonmsg=Left. clid=1"
>>> 
>>> print( resp )
b'notifyclientleftview cfid=1 ctid=0 reasonid=8 reasonmsg=Left. clid=1'
>>> 
```